layouts/index.html

```diff
@@ -20,8 +20,8 @@
     <div class="container">
         <div class="row header">
             <p class="lead">
-                Ozone is a scalable, redundant, and distributed object store for Hadoop. Apart from scaling to billions of objects of varying sizes, Ozone can function effectively in containerized environments such as Kubernetes and YARN. </p> <p class="lead">Applications using frameworks like Apache Spark, YARN and Hive work natively without any modifications.</p>
-            <p class="lead">Ozone is built on a highly available, replicated block storage layer called Hadoop Distributed Data Store (HDDS).
+                Apache Ozone is a scalable, redundant, and distributed object and file store for Hadoop. It supports S3 compatible object APIs as well as a Hadoop Compatible File System implementation. Apart from scaling to billions of objects/files of varying sizes, Ozone can function effectively in containerized environments such as Kubernetes and YARN. </p> <p class="lead">It is built on a highly available, replicated block storage layer called Hadoop Distributed Data Store (HDDS).</p>
```

Comment:
Ozone is a highly scalable, distributed storage for Analytics, Big data and Cloud Native applications. Ozone is optimized for both efficient object store and file system operations. Ozone supports efficient path-based access to files (objects) as well as traditional file system operations like directory rename and delete.

Reply:
Thanks @arp7,  a new patch is uploaded to merge the statements.

Reply:
Thanks @arp7,  a new patch is uploaded to merge the statements.